test/e2e/dynamic_provisioning.go

```diff
@@ -63,4 +63,41 @@ var _ = Describe("Dynamic Provisioning", func() {
 		}
 		test.Run(cs, ns)
 	})
+
+	It("should create multiple PV objects, bind to PVCs and attach all to different pods on the same node", func() {
+		pods := []testsuites.PodDetails{
+			{
+				Cmd: "while true; do echo $(date -u) >> /mnt/test-1/data; sleep 1; done",
+				Volumes: []testsuites.VolumeDetails{
+					{
+						FSType:    "ext3",
```

Comment:
actually specify `FSType` does not make sense, we may remove this parameter and only run on case here?

Reply:
Let's keep it for now